_episodes/15-transferring-files.md

```diff
@@ -113,9 +113,10 @@ bottom of the directory tree rooted at the folder name you provided.
 ```
 {: .bash}
 
-> Caution: For a large directory &mdash; either in size or number of files &mdash; copying
+> ## Caution
+> For a large directory &mdash; either in size or number of files &mdash; copying
 > with `-r` can take a long time to complete.
-{: .warning}
+{: .callout}
```

Comment:
Not sure if this is a pertinent change or not. Doesn't appear to relate to the new snippet library.

Reply:
`{: .warning}` is not actually part of the Carpentries syntax, so this currently does not render (you can see this just above https://carpentries-incubator.github.io/hpc-intro/15-transferring-files/index.html#whats-in-a-)

I'll add this to the other PR

Reply:
https://github.com/carpentries-incubator/hpc-intro/pull/313